### DISCUSSION TITLE
Tracking different fields depending on event

### DISCUSSION THREAD

**Discussion by mustafa0x** [AmbitionEng/django-pghistory#151] [Q&A] [answered]
https://django-pghistory.readthedocs.io/en/3.0.0/event_tracking/#tracking-specific-fields

I'd like to track a few fields of insert/update, and all fields for delete. Is that possible?

**Comment by wesleykendall** [comment, marked as answer]
Yes, use multiple invocations of `@pghistory.track`:

```python
@pghistory.track(
    pghistory.InsetEvent(), pghistory.UpdateEvent(),
    fields=["int_field"],
    model_name="MyTrackedModelInsertUpdateHistory"
)
@pghistory.track(
    pghistory.DeleteEvent(),
    fields=["int_field", "char_field"],
    model_name="MyTrackedModelDeleteHistory"
)
class MyTrackedModel(models.Model):
    int_field = models.IntegerField()
    char_field = models.TextField()
```

Each invocation of `track` is associated with a different event model.

A short section for multiple trackers was added in https://github.com/Opus10/django-pghistory/pull/142
